Src/GoogleApis.Tests/Apis/Upload/ResumableUploadTest.cs

```diff
@@ -790,6 +798,40 @@ public void TestChunkUpload_NotFound_KnownSize()
         }
 
         /// <summary>
+        /// Special case of the upload failure, with a plain-text error message.

```

Comment:
Well, we'd end up with it as HTML - but basically it's "JSON or text". If anything decides to give a non-text error, I suspect ReadContentAsString would fail :(